gvm/protocols/gmpv8.py

```diff
@@ -392,3 +392,153 @@ def modify_credential(
+
+        return self._send_xml_command(cmd)
+
+    def modify_tag(
+        self,
+        tag_id,
+        *,
+        comment=None,
+        name=None,
+        value=None,
+        active=None,
+        resource_action=None,
+        resource_type=None,
+        resource_filter=None,
+        resource_ids=None
+    ):
+        """Modifies an existing tag.
+
+        Arguments:
+            tag_id (str): UUID of the tag.
+            comment (str, optional): Comment to add to the tag.
+            name (str, optional): Name of the tag.
+            value (str, optional): Value of the tag.
+            active (boolean, optional): Whether the tag is active.
+            resource_action (str, optional) Whether to add or remove
+                resources instead of overwriting. One of '', 'add',
+                'set' or 'remove'.
+            resource_type (str, optional): Type of the resources to
+                which to attach the tag. Required if resource_filter
+                or resource_ids is set.
+            resource_filter (str, optional) Filter term to select
+                resources the tag is to be attached to. Required if
+                resource_type is set unless resource_ids is set.
+            resource_ids (list, optional): IDs of the resources to
+                which to attach the tag. Required if resource_type is
+                set unless resource_filter is set.
+
+        Returns:
+            The response. See :py:meth:`send_command` for details.
+        """
+        if not tag_id:
+            raise RequiredArgument("modify_tag requires a tag_id element")
+
+        cmd = XmlCommand("modify_tag")
+        cmd.set_attribute("tag_id", str(tag_id))
+
+        if comment:
+            cmd.add_element("comment", comment)
+
+        if name:
+            cmd.add_element("name", name)
+
+        if value:
+            cmd.add_element("value", value)
+
+        if active is not None:
+            cmd.add_element("active", _to_bool(active))
+
+        if resource_action or resource_filter or resource_ids or resource_type:
+            if not resource_filter and not resource_ids:
+                raise RequiredArgument(
+                    "modify_tag requires resource_filter or resource_ids "
+                    "argument when resource_action or resource_type is set"
+                )
+
+            if not resource_type:
+                raise RequiredArgument(
+                    "modify_tag requires resource_type argument when "
+                    "resource_action or resource_filter or resource_ids "
+                    "is set"
+                )
+
+            _xmlresources = cmd.add_element("resources")
+            if resource_action is not None:
+                _xmlresources.set_attribute("action", resource_action)
+            if resource_filter is not None:
+                _xmlresources.set_attribute("filter", resource_filter)
+            for resource_id in resource_ids or []:
```

Comment:
```suggestion

            for resource_id in resource_ids or []:
```

Reply:
I think Github doesn't see this as addressed because i amended the original commit with it. Not sure why it worked for some changes.